bin/index.js

```diff
@@ -0,0 +1,145 @@
+#!/usr/bin/env node
+var DraftLog = require('draftlog').into(console).addLineListener(process.stdin)
+var SpeedTestNet = require('../')
+
+const DIM = '\033[2m'
```

Comment:
Might it be better to use [`chalk`](https://github.com/chalk/chalk) for consistent color support?